client/core/trade.go

```diff
@@ -683,7 +683,7 @@ func (t *trackedTrade) isSwappable(match *matchTracker) bool {
 func (t *trackedTrade) isRedeemable(match *matchTracker) bool {
 	dbMatch, metaData, proof, _ := match.parts()
 	if match.failErr != nil || len(proof.RefundCoin) != 0 {
-		log.Debugf("Match %v not redeemable: failErr = %v, RefundCoin = %v",
+		log.Tracef("Match %v not redeemable: failErr = %v, RefundCoin = %v",
```

Comment:
Let's just be certain that failErr is logged elsewhere when it is set to something non-nil.

Reply:
Do you mind if I get to this in a subsequent work? I plan to look into this variable more when working on setting missing matches as revoked, would likely be a good place to do this.

Reply:
Sure.  It looks like there's a c.notify for any such error anyway.